Bioinformatic tools for tRNA genes analyses in mitochondrial DNA sequence data

Amphipod-specific_CMs.zip - archive with the amphipod-specific models for searching of  tRNA genes in mt genomes using MiTFi.

script_best_alignment.zip  - archive that contains R script for testing the best sequence alignment, its input and output files. We  developed R script for estimation of the best mode of sequence alignments, which was applied to access the best alignment of tRNA genes in, but it is also suitable for testing of any nucleotide or aminoacid alignments sets used in phylogenetic inferences. 

script_gen_distances.zip -   archive that contains R script estimation of tested tRNA gene sequence resemblance to certain isoacceptor type, its input and output files. We developed R script automating the diagnosis the ancestor tRNA gene coding specificity regardless their anticodon sequence based on genetic distances comparison. 

tRNA_gene_annotation_amphipods.xlsx – The file contains annotation of tRNA genes in mt genomes of amphipods performed by Mitfi using both metazoan and amphipod-specific models.

CM_analysis_metazoan_models.xlsx – The file contains the result of covariance model (cm) scoring analysis of  amphipods tRNA genes based on metazoan models implemented in the Infernal software package (Nawrocki and Eddy 2013).

tRNA_gene_annotation_amphipods.xlsx– The file contains the result of covariance model (cm) scoring analysis of  amphipods tRNA genes performed by Infernal and based on amphipod-specific models.
